tensorboard/webapp/metrics/views/card_renderer/scalar_card_fob_controller.ts

```diff
@@ -74,18 +74,18 @@ export class ScalarCardFobController {
     return this.scale.forward(
       this.minMaxHorizontalViewExtend,
       [0, this.axisSize],
-      this.timeSelection.start.step
+      this.timeSelection?.start.step ?? this.minMaxStep.minStep
     );
   }
 
   getAxisPositionFromEndStep() {
-    if (this.timeSelection.end === null) {
+    if (!this.timeSelection?.end) {
       return null;
```

Comment:
why return null not empty string?

Reply:
I have no idea, you wrote this line of code.

Reply:
oops.